app/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -90,3 +94,49 @@ def test_end_experiment_and_reject_end(
     Dashboard(selenium, kinto_review_url).open().reject()
 
     SummaryPage(selenium, experiment_url).open().wait_for_rejected_alert()
+
+
+@pytest.mark.run_once
+def test_create_new_experiment_approve_check_targeting(
+    selenium,
+    base_url,
+    kinto_url,
+    kinto_review_url,
+    experiment_url,
+    default_data,
+    create_experiment,
+    slugify,
+):
+    default_data.audience.targeting = (
+        BaseExperimentAudienceTargetingOptions.TARGETING_MAC_ONLY
+    )
+    default_data.audience.locale = "English (Canadian)"
+    default_data.audience.countries = "Canada"
+    create_experiment(selenium).launch_and_approve()
```

Comment:
Actually instead of launching/approving just send it to preview and then you don't need to do the whole remote settings flow but it will still show up in the V6 API.